File: src/main/qm/banking/domain/singlepayments/implementation/BalanceBuilder.java
package qm.banking.domain.singlepayments.implementation;

import qm.banking.domain.entities.Account;
import qm.banking.domain.singlepayments.api.Balance;

import java.time.LocalDateTime;

public class BalanceBuilder {
    public static Balance balance(Account account) {
        return new Balance(account.getIBAN(), LocalDateTime.now(), account.getBalance());
    }
}

File: src/main/qm/banking/domain/singlepayments/implementation/BalanceBuilder.kt
package qm.banking.domain.singlepayments.implementation

import qm.banking.domain.entities.Account
import qm.banking.domain.singlepayments.api.Balance
import java.time.LocalDateTime

object BalanceBuilder {
    fun balance(account: Account): Balance {
        return Balance(account.iBAN, LocalDateTime.now(), account.balance)
    }
}